qcodes/instrument_drivers/tektronix/Keithley_2600.py

```diff
@@ -35,26 +114,51 @@ def __init__(self, name, address, channel, **kwargs):
                            get_cmd='source.func',
                            get_parser=float,
                            set_cmd='source.func={:d}',
-                           val_mapping={'current': 0, 'voltage': 1})
+                           val_mapping={'current': 0, 'voltage': 1},
+                           docstring='Selects the output source.')
         self.add_parameter('output',
                            get_cmd='source.output',
                            get_parser=float,
                            set_cmd='source.output={:d}',
                            val_mapping={'on':  1, 'off': 0})
-        # Source range
-        # needs get after set
-        self.add_parameter('rangev',
+        self.add_parameter('nplc',
+                           label='Number of power line cycles',
+                           set_cmd='measure.nplc={:.4f}',
+                           get_cmd='measure.nplc',
+                           get_parser=float,
+                           vals=vals.Numbers(0.001, 25))
+        # volt range
+        # needs get after set (WilliamHPNielsen): why?
```

Comment:
in which way. To be up to date?

Reply:
I only added the "why", the comment was already in the driver.

Reply:
Fair enough I guess I'm not the only one who thinks this is unclear